lib/taskana-core/src/test/java/acceptance/task/TransferTaskAccTest.java

```diff
@@ -138,7 +138,7 @@ void should_ThrowException_When_TaskToTransferDoesNotExist() {
 
     ThrowingCallable call =
         () -> {
-          taskService.transfer("Invalid", "WBI:100000000000000000000000000000000006");
+          taskService.transfer("Invalid", "WBI:100000000000000000000000000000000005");
```

Comment:
For the Workbasket previously used the User does not have a READ permission. Before it had not been a problem because there was another order of checks and thus of the Exception thrown. The renewed implementation first checks the Workbasket Preconditions of the target Workbasket and then the Preconditions for a Task to be transferred. It is consistent with our bulkTransfer.